internal/xds/rbac/matchers.go

```diff
@@ -344,7 +345,8 @@ func newRemoteIPMatcher(cidrRange *v3corepb.CidrRange) (*remoteIPMatcher, error)
 }
 
 func (sim *remoteIPMatcher) match(data *rpcData) bool {
-	return sim.ipNet.Contains(net.IP(net.ParseIP(data.peerInfo.Addr.String())))
```

Comment:
Not sure what this old redundant `net.IP` cast was about, but now it's actually necessary.  :)